plugins/weather.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+#  weather.rb
+#  Author: William Woodruff
+#  ------------------------
+#  A Cinch plugin that provides APIXU weather interaction for yossarian-bot.
+#  ------------------------
+#  This code is licensed by William Woodruff under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require 'bundler/setup' # needed for apixu
+require 'apixu'
+require_relative "yossarian_plugin"
+
+class Weather < YossarianPlugin
+  include Cinch::Plugin
+  use_blacklist
+
+  KEY = ENV["APIXU_API_KEY"]
+
+  def usage
+    "!w <location> - Get the weather at <location>. Alias: !weather."
+  end
+
+  def match?(cmd)
+    cmd =~ /^(!)?(weather)|(^w)$/
+  end
+
+  match /w (.+)/, method: :weather, strip_colors: true
+  match /weather (.+)/, method: :weather, strip_colors: true
+
+  def weather(m, location)
+    if KEY
+      begin
+        client = Apixu::Client.new KEY
+        hash = client.current location
+      rescue Apixu::Errors::Request => e
+        if e.code == Apixu::Errors::NO_LOCATION_FOUND_FOR_QUERY
+          m.reply "Nothing found for location \'#{location}\'.", true
+        else
+          m.reply e, true
+        end
+      else
+        loc = hash["location"]["name"]
+        loc += ", #{hash["location"]["region"]}" if hash["location"]["region"].length > 0
```

Comment:
its an empty string, see the response for `1.1.1.1`:
![image](https://user-images.githubusercontent.com/15038414/63791170-284d1e80-c8fb-11e9-9d74-0758dd46f215.png)
I changed it to `unless .. empty?` now though because rubocop suggested it